src/server/http/orchestrator/orchestrator.types.ts

```diff
@@ -0,0 +1,120 @@
+import { z } from 'zod';
+import { trainingDetailsParams } from '~/server/schema/model-version.schema';
+
+export namespace Orchestrator {
+  export type Job<TResult = unknown> = { jobId: string; result: TResult };
+  export type JobResponse<TJob = Job> = { token: string; jobs: TJob[] };
+  export type JobQueryParams = { wait?: boolean };
```

Comment:
i'm not sure if we want to explicitly type these or keep them generic, but this also can accept `force?: boolean`

Reply:
I believe explicitly typing the query params helps to know what's available to send since there's no actual docs for the orchestrator API atm 😛